res/windows/wix/space-acres.wxs

```diff
@@ -216,6 +216,9 @@
                         <Component Id='gtk4_pangocairo_1.0_0.dll' Guid='9f0d20f0-5267-4e40-a081-2d061774b9f9'>
                             <File Id='pangocairo_1.0_0.dll' Name='pangocairo-1.0-0.dll' DiskId='1' Source='$(var.CargoTargetDir)\wix\gtk4\bin\pangocairo-1.0-0.dll' />
                         </Component>
+                        <Component Id='gtk4_pangoft2_1.0_0.dll' Guid='965875b0-4b2b-4117-b570-efb009add1e2'>
```

Comment:
This is not enough, it also needs to be included below, see other examples

Reply:
Fixed! Are there step-by-step instructions for this that we can link to in the developer README?

Reply:
Not really, just look at what is in there and modifying by analogy. CI checks only catch so much.